core/src/test/java/io/questdb/test/griffin/engine/functions/math/AddIntFunctionFactoryTest.java

```diff
@@ -25,32 +25,25 @@
 package io.questdb.test.griffin.engine.functions.math;
 
 import io.questdb.griffin.FunctionFactory;
-import io.questdb.griffin.SqlException;
-import io.questdb.test.griffin.engine.AbstractFunctionFactoryTest;
 import io.questdb.griffin.engine.functions.math.AddIntFunctionFactory;
-import io.questdb.std.Numbers;
+import io.questdb.test.griffin.engine.AbstractFunctionFactoryTest;
 import org.junit.Test;
 
 public class AddIntFunctionFactoryTest extends AbstractFunctionFactoryTest {
 
     @Test
-    public void testLeftNull() throws SqlException {
-        call(Numbers.INT_NULL, 10).andAssert(Numbers.INT_NULL);
-    }
-
-    @Test
-    public void testOverflow() throws SqlException {
```

Comment:
@bluestreak01 would new treatment of int overflow in addition functions be a breaking change? I've removed this test

Reply:
Yep, it would be a mild breaking change. I don’t think anyone would miss random numbers as a result of overflow